src/main/java/org/venice/beachfront/bfapi/model/SceneStatus.java

```diff
@@ -0,0 +1,5 @@
+package org.venice.beachfront.bfapi.model;
+
+public class SceneStatus {
```

Comment:
What is the purpose of this class? Can we re-use the static status strings in Job.java?

Reply:
Or I suppose you'll use the status in Scene.java - since those are specific to Scenes